credentials-generator/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Ok being an independent tool, but let it inherit from root `pom.xml` so that all versions can be set there.

Reply:
Initially was my intention but have issues while building, since the parent pom packaging is `jar`.

Reply:
Understood.
So either change the project structure into parent pom + 2 child modules or keep the existing structure with a single JAR module without splitting the generator as an external tool